### DISCUSSION TITLE
Can I assing a specific image for a specific user/group and/or IP?

### DISCUSSION THREAD

**Discussion by RaulBustamante** [ltsp/ltsp#603] [Q&A]
Hello,

I have 3 different images, all for 3 different departments with different activities and applications. When booting up by IPXE it shows all 3 images to select the one that will be used. Is it possible to have this assigned to the specific user or Client IP instead? To avoid them selecting the incorrect image? thank you.

**Comment by alkisg** [comment]
Yes, see the [ltsp.conf man page](https://ltsp.org/man/ltsp.conf/), and specifically the `DEFAULT_IMAGE=the-image-you-want` parameter.

**Reply by perlland** [reply]
It didn't work.

**Reply by perlland** [reply]
I also wanted the solution to this problem  where I can assign a specific image to a specific user using MacID, but this solution didn't work.

**Comment by alkisg** [comment]
Did you run `ltsp ipxe` after the change?

**Reply by perlland** [reply]
ltsp initrd is what i did

**Reply by perlland** [reply]
even after running ltsp ipxe it didn't work

**Comment by alkisg** [comment]
Feel free to come to the [chat room](https://ltsp.org/advanced/chat-room/) for live support if you wish.

**Comment by vsuojanen** [comment]
I have similar questions. I hope you don't mind joining here.

I know it is possible to have one PXE boot server and several  image/application servers.  
If I have a new LTSP image tested initially on different server LTSP development network and want to join the new server as is in same network with existing LTSP production server (and shutdown dnsmasq proxydhcp  in the new server before running it together with another LTSP server in same LTSP network).

having different client image and apps for some client is the correct path something like
1) manually new client MAC specific sections in iPXE (ltsp.ipxe)
2) manually or with DEFAULT_IMAGE=the-image-you-want (ltsp.conf) the new image section to ltsp.ipxe
3) possible different client kernel params (ltsp.conf) to ltsp.ipxe ? i don't know if this is needed but I think the kernel, ltsp.img and initrd should exist in the boot server right?

How does this sound ?
I might pop to chat also

**Comment by alkisg** [comment]
@vsuojanen I don't think I understood the question well enough.
From what I understood, you want to find the best way to organize multiple images and servers, with different kernel parameters for each client.

I think it would be best to open a separate discussion for that, but please try to provide more details.
For example, a single server may have many images; and even if the clients boot from it, they users may log in to a different SSH_SERVER; and you may also use proxyDHCP in all the servers for DHCP load balancing/failover; etc etc, there are so many options that you need to be very specific, in order to get good advice.

**Reply by perlland** [reply]
after running ltsp ipxe my issue is resolved thanks
